e2e-tests/tests/remove-element.spec.ts

```diff
@@ -10,11 +10,17 @@ const expectedPage = fs.readFileSync(
 studioTest("can remove a container component", async ({ studioPage }) => {
   const containerPreviews = studioPage.preview.getByText("I'm a container:");
   const childPreviews = studioPage.preview.getByText("false");
+  const componentTreeSection = studioPage.componentTreeSection;
+
   await expect(containerPreviews).toHaveCount(1);
   await expect(childPreviews).toHaveCount(1);
-  await studioPage.removeElement("Container");
+  await componentTreeSection.takeScreenshot();
+  await componentTreeSection.removeElement("Container");
+  await componentTreeSection.takeScreenshot();
   await expect(containerPreviews).toHaveCount(0);
   await expect(childPreviews).toHaveCount(0);
+
+  // TODO: Scope this to Screenshot of Preview pane.
   await studioPage.takePageScreenshotAfterImgRender();
```

Comment:
Are we planning on changing most of our currently un-scoped screenshots to be scoped to the preview panel?

Reply:
I think a number of un-scoped screenshots will become scoped to the preview panel, yes. There will also be a few situations in which we replace the un-scoped screenshot with the screenshot of one of the other Section models.